src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -1106,6 +1102,10 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         new ItemStack[] {SlimefunItems.GOLD_24K, null, SlimefunItems.GOLD_24K, new ItemStack(Material.LEATHER), SlimefunItems.GILDED_BACKPACK, new ItemStack(Material.LEATHER), SlimefunItems.GOLD_24K, null, SlimefunItems.GOLD_24K})
         .register(plugin);
 
+        new ContainmentBackpack(27, categories.usefulItems, SlimefunItems.CONTAINMENT_BACKPACK, RecipeType.ENHANCED_CRAFTING_TABLE,
+        new ItemStack[] {SlimefunItems.REINFORCED_CLOTH, SlimefunItems.COOLING_UNIT, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.BACKPACK_LARGE, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.REINFORCED_CLOTH})
```

Comment:
Maybe add some lead ingots to the recipe, in place of a couple reinforced cloth. This is just a suggesstion though.